t/construct_publish_urls.t

```diff
@@ -95,17 +97,17 @@ $uri = Mojo::URL->new($urls[0]->{url});
 is $uri->path, '/publish/demo/demo/0/sandbox/0/%22test3%22';
 is $uri->query->param('ear'), '"True"';
 is $uri->query->param('seqn'), '12345';
-is_deeply decode_json($uri->query->param('meta')), { "stuff" => [] };
-is_deeply decode_json($uri->query->param('ortt')), {
+is_deeply $json->decode(Encode::decode_utf8($uri->query->param('meta'))), { "stuff" => [] };
+is_deeply $json->decode(Encode::decode_utf8($uri->query->param('ortt'))), {
```

Comment:
I believe that Mojo::URL's `->query->param` method already decodes UTF-8 bytes into Unicode strings: https://metacpan.org/pod/Mojo::Parameters#charset

> Charset used for encoding and decoding parameters, defaults to UTF-8.

So this line probably doesn't have to.